src/index.tsx

```diff
@@ -24,12 +24,7 @@ export type {
 	UiSchema,
 	FormValidation,
 } from './components/Form';
-export type {
-	FileWidget,
```

Comment:
I recommend keeping the export on the fileWidget, as it might be needed in the future.